src/components/network/network-table.js

```diff
@@ -360,6 +406,21 @@ const NetworkTable = (props) => {
         ]
     );
 
+    const onTabChange = useCallback(
+        (index) => {
+            console.debug('onTabChange from ' + tabIndex + ' to ' + index);
```

Comment:
I don't think we need to log the index of the tab when we change it

Reply:
ok, I will remote it